lib/logic/list-controller.js

```diff
@@ -10,30 +10,23 @@ function ListController(dispatcher, httpApi, controller) {
 }
 
 ListController.prototype.getOpenPullRequests = function() {
-    var self = this;
     Promise.try(() =>
-        self.httpApi.getUser()
+        this.httpApi.getUser()
             .then((userData) =>
                 Promise.try(() =>
-                    Promise.join(
-                        self.httpApi.getOpenPullRequests(userData.body.login, "author")
-                            .then((prs) => self._fillComments(prs))
-                            .then(self._loadMyPulls),
-
-                        self.httpApi.getOpenPullRequests(userData.body.login, "assignee")
-                            .then((prs) => self._fillComments(prs))
-                            .then(self._loadAssignedPulls),
-
-                        (myPRs, assignedPRs) =>
-                            self.dispatcher.dispatch(
-                                new triggers.PullRequestListTrigger(
-                                    myPRs, assignedPRs
-                                )
-                            )
-                    )
+                    this.httpApi.getOpenPullRequests(userData.body.login, "involves")
+                        .then((prs) => this._fillComments(prs))
+                        .then((prs) => this._loadPulls(
+                            prs,
+                            apiMapper.getUserFromGithubApi(userData.body))
+                        )
+                        .then((prs) =>
+                            this.dispatcher.dispatch(
+                                new triggers.PullRequestListTrigger(prs)
+                            ))
```

Comment:
Please be strict with yourself on bracket matching. It's really hard to see if brackets are correct or sane here. I would really like to enforce pairing the close bracket at the same level of indentation as the open bracket. Take `:19 -> :22` as an example:

```
// You have
.then((prs) => this._loadPulls( // 2 open brackets
  prs,
  apiMapper.getUserFromGithubApi(userData.body)) // close 1
) // close 1

// I would prefer
.then((prs) => this._loadPulls( // 2 open brackets
  prs,
  apiMapper.getUserFromGithubApi(userData.body)
)) // 2 closed brackets
```
When they are being chained like this it's even more important.